html/README.md

```diff
@@ -164,10 +165,25 @@ If you must include the attribute's value, and **you don't need to**, follow thi
 <input type="checkbox" value="1" checked>
 
 <select>
-  <option value="1" selected>1</option>
+    <option value="1" selected>1</option>
 </select>
 ```
 
+## IDs vs classes
+
+Always use classes for styles. When possible, use IDs when you need to target an element in JavaScript. This makes 
+elements used in JavaScript very easy to spot.
+
+```html
+<a class="...">
+    I am a styled link
+</a>
+
+<form class="..." id="...">
```

Comment:
The `data-` attributes are also used for parameterisation (often). I'm not saying we should use IDs exclusively, but I see some cases where an ID is more appropriate because it is more explicit.